source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -127,6 +132,17 @@ corresponds to the format of the example you want to see:
            "numViews": NumberLong("36520312")
          }
 
+   .. tab:: Strict
+      :tabid: strict-format
+
+      .. code-block:: json
+
+         {
+           "_id:": { "$oid": "573a1391f29313caabcd9637" },
+           "createdAt": { "$date": "2020-09-30T18:22:51.648Z" },
```

Comment:
This isn't what `STRICT` produces.  From my tests, this:

```
        System.out.println(doc.toJson(JsonWriterSettings.builder().outputMode(JsonMode.EXTENDED).build()));
        System.out.println(doc.toJson(JsonWriterSettings.builder().outputMode(JsonMode.RELAXED).build()));
        System.out.println(doc.toJson(JsonWriterSettings.builder().outputMode(JsonMode.STRICT).build()));
```

outputs

```
{"d": {"$date": {"$numberLong": "1647465431487"}}}
{"d": {"$date": "2022-03-16T21:17:11.487Z"}}
{"d": {"$date": 1647465431487}}
```

Reply:
My test code was incorrect, will add the corrected version to the repo so future authors (hopefully) don't make the same mistake.